tools/seahorn/seahorn.cpp

```diff
@@ -316,8 +316,8 @@ int main(int argc, char **argv) {
   }
   pass_manager.add(new seahorn::RemoveUnreachableBlocksPass());
 
-  pass_manager.add(seahorn::createPromoteMallocPass());
   pass_manager.add(seahorn::createPromoteVerifierCallsPass());
+  pass_manager.add(seahorn::createPromoteMallocPass());
```

Comment:
hmm... this might be masking the problem of mapping `free` to `sea.free`. The order of these two passes should not matter in general...

Reply:
removed change... was debugging and forgot to remove.